tests/smoke.test.js

```diff
@@ -8,7 +8,8 @@ describe('smoke test', () => {
     it('should wait for elements if not found immediately', () => {
         browser.waitForElementScenario()
         const elem = $('elem')
-        assert.equal(elem.isDisplayed(), true)
+        //Element will be found
+        elem.click()
```

Comment:
If the click doesn't throw up... we are good!

Reply:
We should have an additional test for this use case.

Reply:
I think done?

Reply:
Or did you mean you want me to write a test specifically for displayed?